src/actions/SendConfirmationActions.js

```diff
@@ -23,7 +23,7 @@ import s from '../locales/strings.js'
 import { addToFioAddressCache, recordSend } from '../modules/FioAddress/util'
 import { getExchangeDenomination as settingsGetExchangeDenomination } from '../modules/Settings/selectors.js'
 import { type AuthType, getAuthRequired, getSpendInfo, getSpendInfoWithoutState, getTransaction } from '../modules/UI/scenes/SendConfirmation/selectors'
-import { convertCurrencyFromExchangeRates, getExchangeRate, getSelectedCurrencyCode, getSelectedWallet, getSelectedWalletId } from '../modules/UI/selectors.js'
+import { convertCurrencyFromExchangeRates, getExchangeRate } from '../modules/UI/selectors.js'
```

Comment:
should we move this methods to util file or somethins?

Reply:
I think that would make sense eventually, but for now the goal was to just get rid of the selectors.

Reply:
ok